default.env

```diff
@@ -2,26 +2,21 @@
 TZ=Europe/Berlin  
 ALWAYS_UPDATE_ON_START=true
 MULTITHREAD_ENABLED=true
-COMMUNITY_SERVER=true
+STEAMCMD_VALIDATE_FILES=true
+
+### Backups
 BACKUP_ENABLED=true
 BACKUP_CRON_EXPRESSION=0 * * * *
 BACKUP_RETENTION_POLICY=false
 BACKUP_RETENTION_AMOUNT_TO_KEEP=30
-STEAMCMD_VALIDATE_FILES=true
-# Change this to manual if you want to edit your configs yourself
-SERVER_SETTINGS_MODE=auto 
-WEBHOOK_ENABLED=false
-WEBHOOK_URL="YOUR-URL-IN-HERE"
-WEBHOOK_START_TITLE="Server is starting"
-WEBHOOK_START_DESCRIPTION="The gameserver is starting"
-WEBHOOK_START_COLOR="2328576"
-WEBHOOK_STOP_TITLE="Server has been stopped"
-WEBHOOK_STOP_DESCRIPTION="The gameserver has been stopped"
-WEBHOOK_STOP_COLOR="7413016"
-WEBHOOK_INFO_TITLE="Info"
-WEBHOOK_INFO_DESCRIPTION="This is an info from the server"
-WEBHOOK_INFO_COLOR="2849520"
+
+### Server Settings
+# Change this to manual if you want to edit your configs directly on file
```

Comment:
Looks weird in the review, need to look at it later in the IDE, dont want to have to much documentation in the env file, thats why we have readmes for all stuff. No need for dublicate content and confusing people. Because now one is explained, why not all of them?! When all of them are explained, why have a readme?! You get my point 😄

Reply:
Was already present, just categorized settings if people download the .env and want to change it's better to find settings.